src/report/report.py

```diff
@@ -595,6 +598,9 @@ def _get_issue_labels(
         # Add any additional labels
         if jira_additional_labels:
             labels.extend(jira_additional_labels)
+        if jira_additional_labels_filepath:
+            with open(jira_additional_labels_filepath, "r") as file:
+                labels.extend(line.strip() for line in file if line.strip())
```

Comment:
why is `if line.strip()` needed?

Reply:
@rnetser it's not needed. I originally was using it to filter any empty lines or lines with only whitespace. After some further testing, I found it works without. Removed.